automatic/cmake/update.ps1

```diff
@@ -31,13 +31,27 @@ function global:au_GetLatest {
     $url32_portable = $allZips | Where-Object { $_ -match "$version-$re32" }
     $url64_portable = $allZips | Where-Object { $_ -match "$version-$re64" }
 
-    $streams.Add($versionTwopart, @{
-        Version = $version
-        URL32_i = [uri]$_
-        URL64_i = [uri]$url64
-        URL32_p = [uri]$url32_portable
-        URL64_p = [uri]$url64_portable
-      })
+    if ($streams.ContainsKey($versionTwoPart)) {
+        # Prefer release over prerelease if both are available for download
+        if ($streams.$versionTwoPart.Version.contains("-") -and !$version.contains("-")) {
+            $streams.Remove($versionTwopart)
+            $streams.Add($versionTwopart, @{
```

Comment:
You do not need to remove the existing version first.

Instead do

```suggestion
            $streams[$versionTwopart] = @{
```

Reply:
Fixed up.